Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/Aquality.Selenium.Template.SpecFlow.csproj

```diff
@@ -28,6 +29,9 @@
     <None Update="allureConfig.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Update="Resources\aqualityTracking.json">
```

Comment:
Do we have ability to copy the config template after adding a nuget package like Allure does?

Reply:
Actually, it does, But we cannot use copied file. We have to create our own. That's why I've added `aqualityTracking.Template.json` to AqualityTracking,SpecFlowPlugin package. If you open solution in VS you'll see this file copied from nuget automatically.